src/main/java/com/answerdigital/answerking/response/CategoryResponse.java

```diff
@@ -24,10 +26,10 @@ public class CategoryResponse {
     private List<Long> products;
 
     @JsonProperty(access = JsonProperty.Access.READ_ONLY)
-    private String createdOn;
+    private LocalDateTime createdOn;
 
     @JsonProperty(access = JsonProperty.Access.READ_ONLY)
-    private String lastUpdated;
+    private LocalDateTime lastUpdated;
```

Comment:
Just wondering if we can keep the string in here as in the API spec it says that the return type should be string